zingolib/src/wallet/data.rs

```diff
@@ -29,8 +29,8 @@ use zingoconfig::{ChainType, MAX_REORG};
 use super::keys::unified::WalletCapability;
 use super::traits::{self, DomainWalletExt, ReadableWriteable, ToBytes};
 
-pub const COMMITMENT_TREE_DEPTH: u8 = 32;
-pub const MAX_SHARD_DEPTH: u8 = 16;
+pub const COMMITMENT_TREE_LEVELS: u8 = 32;
```

Comment:
I'm...ambivalent towards this change. Intuitive parlance for me when talking about how many levels up/down something is would be 'levels high/level deep'...if this is clearer to you I'm fine with it, though.